v03_pipeline/lib/misc/validation.py

```diff
@@ -11,9 +24,43 @@ class SeqrValidationError(Exception):
     pass
 
 
+def get_validation_dependencies(
+    dataset_type: DatasetType,
+    reference_genome: ReferenceGenome,
+    callset_path: str,
+    skip_validation: bool,
+    skip_check_sex_and_relatedness: bool,
+    **_: Any,
+) -> dict[str, hl.Table]:
+    deps = {}
+    if skip_validation or not dataset_type.can_run_validation:
```

Comment:
this function is only called once and its in an `if not self.skip_validation and self.dataset_type.can_run_validation:` block so I think this check is redundant

Reply:
👍 I would love to reduce the number of places we have this check 😬